assets/javascripts/discourse/components/solved-unaccept-answer-button.gjs

```
@@ -67,9 +37,28 @@ export default class SolvedUnacceptAnswerButton extends Component {
     }
   }
 
+  @action
+  async unacceptAnswer() {
+    const post = this.args.post;
+
+    this.saving = true;
+    try {
+      await unacceptPost(post);
+    } finally {
+      this.saving = false;
+    }
```

should we surface an error when it fails?